### DISCUSSION TITLE
Getting "lines too long" error even with all 'long lines' within a disable/enable

### DISCUSSION THREAD

**Discussion by Spinner4000** [streetsidesoftware/vscode-spell-checker#1348] [Q&A] [answered]
Getting "lines too long" error even though all 'long lines' are between disable/enable.
Oddly enough, this was not happening until recently (i.e. same file, same disable/enable placements was being spell checked)
Not exactly sure when the issues started, but I first noticed it yesterday.
Have gone back to older versions of the file (to check if it caused by a change I made), and it's happening there also. 

Has something changed recently that could lead to this symptom?
If not, any clues as how to solve?

**Comment by Jason3S** [comment, marked as answer]
Please see #1256.

The setting was added in the 2.0 release of the spell checker to help detect generated files. You can change the value in preferences:
VS Code Settings -> spell block line

![image](https://user-images.githubusercontent.com/3740137/134811096-f04579c9-9174-490e-b692-b8a0399a124f.png)

**Reply by Spinner4000** [reply]
Thank you. I saw the related error and looked at that setting.
However, this seemed like it had been there for a while (must have been wrong about that).
Also, I assumed it wouldn't break on long lines that are between disable/enable (which it also seems I was wrong about).

Have put the line length to 1,000,000 and that solves my issue. But I wonder 'what are the ramifications of that?'
Could I ask what those might be?

Also, perhaps consider not applying that test to anything between disable/enable.
After all, you won't be checking those lines, so why test length of them?

Lastly, great tool. For which I thank you. : )
